pkg/reconciler/buildrun/resources/sources/bundle.go

```diff
@@ -31,6 +37,11 @@ func AppendBundleStep(
 	bundleStep.Container.Args = []string{
 		"--image", source.BundleContainer.Image,
 		"--target", fmt.Sprintf("$(params.%s-%s)", prefixParamsResultsVolumes, paramSourceRoot),
+		"--result-file-image-digest",
+		fmt.Sprintf(
+			"$(results.%s-source-%s-bundle-image-digest.path)",
+			prefixParamsResultsVolumes, name,
+		),
```

Comment:
Nit: Could fit into one line.
```suggestion
		"--result-file-image-digest", fmt.Sprintf("$(results.%s-source-%s-bundle-image-digest.path)", prefixParamsResultsVolumes, name),
```